<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddModel</class>
 <widget class="QWidget" name="AddModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>234</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加模型</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 名    称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DIR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 文件位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DIRBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>29</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>选 取</string>
   </property>
  </widget>
  <widget class="QLabel" name="comment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 描    述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commentBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirmBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确 认</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveDIR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> 保存位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveDIRBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSelectBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>109</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>选 取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
